sympy/concrete/summations.py

```diff
@@ -670,6 +673,99 @@ def is_absolutely_convergent(self):
         Sum.is_convergent()
         """
         return Sum(abs(self.function), self.limits).is_convergent()
+    @staticmethod
+    def is_expr_convergent(expr):
+        """
+        Checks for convergence of an expression based on properties of convergence and divergence.
+        Parameters
+        ----------
+        expr : TYPE
```

Comment:
Please replace `TYPE` and `DESCRIPTION` here and below with actual types and descriptions (or remove it).